library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java

```diff
@@ -727,6 +734,15 @@ public Builder roundRectRadius(int roundRectRadius) {
         }
 
         /**
+         * @param animateFocus true if Focus should be animated, false otherwise
+         * @return Builder
+         */
+        public Builder animateFocus(final boolean animateFocus) {
```

Comment:
Can you refactor method name to `disableFocusAnimation()` and remove boolean parameter? Because animation is enabled by default. Using `.animateFocus(true)` is useless.